packages/modules/pgr/src/pages/citizen/Rating/SelectRating.js

```diff
@@ -46,7 +45,7 @@ const SelectRating = ({ parentRoute }) => {
       },
       {
         type: "checkbox",
-        label: t("CS_FEEDBACK_WHAT_WAS_GOOD"),
+        label: "CS_FEEDBACK_WHAT_WAS_GOOD",
```

Comment:
again why translation is removed?

Reply:
the translation is already handled in the Rating Card. It's creating an error as the function is expecting "CS_FEEDBACK_WHAT_WAS_GOOD" but getting translated text.
```
  if (input.type === "checkbox") {
      return (
        <React.Fragment key={index}>
          <CardLabel>{t(input.label)}</CardLabel>
          {input.checkLabels &&
            input.checkLabels.map((label, index) => <CheckBox key={index} name={input.label} label={label} inputRef={register} />)}
        </React.Fragment>
      );
```
    }